content/_data/changelogs/weekly.yml

```diff
@@ -1066,6 +1066,29 @@
       pull: 2999
     # pull: 2997 too minor
     # pull: 2989 too minor / don't want to give impression that this use is supported
+- version: "2.78"
+  date: 2017-09-10
+  changes:
+    - type: rfe
+      message: >
+        Moved Jenkins agent runtime to <code>agent.jar</code> file name, and deprecate (but still support) use of legacy <code>slave.jar</code>.
```

Comment:
Nothing really moves in the Jenkins core (there is no slave.jar or agent.jar packaged), there is only change in the suggested names and in the URL. But probably it's fine for the user Javadoc

Reply:
Turns out *moving* a file around doesn't change its location on disk either 😄 So, close enough.